src/config/constants/trading-competition/easter.ts

```diff
@@ -0,0 +1,186 @@
+export enum Tiers {
+  GOLD = 'Gold',
+  SILVER = 'Silver',
+  BRONZE = 'Bronze',
+  PURPLE = 'Purple',
+  TEAL = 'Teal',
+}
+
+export interface Achievement {
+  champion?: number
+  teamPlayer?: number
+  trophy: number
+}
+
+export interface Rank {
+  rank: string
+  tier: Tiers
+  cakePrizeInUsd: number
+  achievements: Achievement
+  hasNft: boolean
+}
+
+interface Config {
+  [key: string]: Rank[]
+}
```

Comment:
This is the first competition so I am keeping these types in here. I imagine though that future competitions will look similar. Once we hash out a good form then we can moves these up to the global type object.